dts/bindings/dma/dma-controller.yaml

```diff
@@ -33,3 +33,13 @@ properties:
       type: int
       required: false
       description: Number of DMA request signals supported by the controller.
+
+    dma-offset:
+      type: int
+      required: false
+      description: >
+        offset in the table of channels when mapping to a DMAMUX
+        for 1st dma instance offset is 0
+        for 2nd dma instance, offset is the nb of dma channels of the 1st dma instance
+        for 3rd dma instance, offset is the nb of dma channels of the 1st
+        dma instance plus the nb of dma channels of the 2nd dma instance, etc
```

Comment:
Is `dma-offset` really the right name?  Would `dmamux-offset` maybe be better? Or `dmamux-channel-offset` ?

Reply:
it could be more confusing to include dmamux in the dma binding

Reply:
Well "confusing" is the right word to describe the stm32dma :-) 

I just thought it would match the code `config->streams[i].mux_channel = i + config->offset;` more closely because there it is also the mux_channel.

Reply:
Or another thought, would a `dma-offset = <0, 7>;` in the dmamux driver be a solution? Although even if it is, it probably just reverses the problem you trying to fix, because than you'll need the dmamux device in the dma driver.

Reply:
I tried this or something similar : to include in the dmamux DTS, the references of the dma offset (though a phandle of dma-masters = <&dma, &dma2> or  directly the offset of each dma). However, the table is set in the **dma init**, at a time the dmamux is not yet known init'ed. 
There is a cross dependency from each dma  channels (dma_stm32_stream) [from 0 to 6]  being link to a dmamux channel [from 0 to 13] and the dmamux channels (dmamux_stm32_channel) referring to their dma instance. There is no simple config to pass info through. 
With this dma-offset in the dma DT, the value is defined where it is used.

Reply:
Yeah that is what I thought, so this is kind of the the best solution that can be done. The name is no show stopper for me,  even though I think that a name like `dmamux-channel-offset` makes it clearer that it has something to do with the dmamux. But I leave that up to you, closing this review as resolved.

Reply:
I fell like @lowlander that `dmamux-channel-offset` would be better to indicate this offset is linked to dmamux, and implicitly developer can found information in DMAMUX chapter (and not in DMA chapter).